applications/luci-app-samba/luasrc/model/cbi/samba.lua

```diff
@@ -13,9 +13,13 @@ s:tab("template", translate("Edit Template"))
 s:taboption("general", Value, "name", translate("Hostname"))
 s:taboption("general", Value, "description", translate("Description"))
 s:taboption("general", Value, "workgroup", translate("Workgroup"))
-s:taboption("general", Value, "homes", translate("Share home-directories"),
+h = s:taboption("general", Flag, "homes", translate("Share home-directories"),
         translate("Allow system users to reach their home directories via " ..
                 "network shares"))
+h.rmempty = false
+h.default = "0"
+h.enabled = "1"
```

Comment:
This looks overly complicated. 0 and 1 are the uci values for enabled/disabled for a Flag.

I think that this could be replaced by just one line:
```
h.default = "0"
h.enabled = "1"
h.disabled = "0"
```
Proposal:
`h.default = h.disabled`

But please test it first... That was just out of my head.